spec/menu_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'menu'
+
+RSpec.describe Menu do
+  let (:chips_menu) do
+    menu = Menu.new()
+    menu.add_dish('Chips', 10)
+    break menu
+  end
+ 
+  describe '#add_dish' do
+    context 'when dish is not yet in menu' do
+      it 'adds a dish to the menu' do
```

Comment:
I guess I am still trying to find my balance between DRY and DAMP in testing. How much setup would I extract out of an 'it' block and how much would I keep inside...

Reply:
It's a difficult balance but it gets easier with practice. Personally I would pull out as much common setup as possible, but keep anything that is relevant to that specific test in the 'it' block for readability. So because this test is testing adding a dish I could keep the adding of the dish within the test.